packages/lab/src/overlay/OverlayPanelCloseButton.tsx

```diff
@@ -0,0 +1,35 @@
+import { forwardRef } from "react";
+import clsx from "clsx";
+import { Button, ButtonProps, makePrefixer } from "@salt-ds/core";
+import { useWindow } from "@salt-ds/window";
+import { useComponentCssInjection } from "@salt-ds/styles";
+import { CloseIcon } from "@salt-ds/icons";
+import overlayPanelCloseButtonCss from "./OverlayPanelCloseButton.css";
+
+const withBaseName = makePrefixer("saltOverlayPanelCloseButton");
+
+export const OverlayPanelCloseButton = forwardRef<
+  HTMLButtonElement,
+  ButtonProps
+>(function OverlayPanelButton({ className, ...rest }, ref) {
+  const targetWindow = useWindow();
+  useComponentCssInjection({
+    testId: "salt-overlay-panel-close-button",
+    css: overlayPanelCloseButtonCss,
+    window: targetWindow,
+  });
+
+  return (
+    <div className={clsx(withBaseName(), className)}>
+      <Button
```

Comment:
In our Button accessibility we say to add a tooltip to buttons with just an icon - is the close button is an exception or should this also have a tooltip ?

Reply:
I imagine this isn't required for something as universally recognisable as a close button. The aria-label is there for accessibility. Maybe the button guidance is too prescriptive.

Reply:
yeh agreed i don't think it is needed